manifest.php

```diff
@@ -43,7 +43,9 @@
     'managementRole' => 'http://www.tao.lu/Ontologies/generis.rdf#taoCeManager',
     'acl' => array(
         array('grant', 'http://www.tao.lu/Ontologies/TAO.rdf#BackOfficeRole', array('ext' => 'taoCe', 'mod' => 'Main', 'act' => 'index')),
-        array('grant', 'http://www.tao.lu/Ontologies/TAO.rdf#BackOfficeRole', array('ext' => 'taoCe', 'mod' => 'Home'))
+        array('grant', 'http://www.tao.lu/Ontologies/TAO.rdf#BackOfficeRole', array('ext' => 'taoCe', 'mod' => 'Home')),
+        array('grant', 'http://www.tao.lu/Ontologies/TAO.rdf#BaseUserRole', array('ext'=>'taoCe', 'mod' => 'Main', 'act' => 'rootEntry')),
```

Comment:
Actually, BaseUserRole includes AnonymousRole. So basically, I think there is no necessity to explicitly grand the BaseUserRole, but just the AnonymousRoule. Indeed, BaseUserRole will implicitly inherits AnymousRole grants.